src/implementations/vanilla/components/select/select.js

```diff
@@ -889,14 +902,21 @@ export class Select {
         break;
 
       case 'ArrowRight':
-      case 'Tab':
         this.clearAllButton.nextSibling.focus();
         break;
 
       case 'ArrowUp':
         this.optionsContainer.lastChild.querySelector('input').focus();
         break;
 
+      case 'Tab':
+        if (e.shiftKey) {
+          this.optionsContainer.lastChild.querySelector('input').focus();
```

Comment:
there is one situation where this breaks: if you have selected at least one item and then you use the search and no result is found, then this container becomes the last child of the optionsContainer and it doesn't contain any input, leading to an error for the undefined input. Same happens if you use arrowUp